client/src/components/timeline-entry-components/card-template-component/index.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import Button from "@mui/material/Button";
+import {
+  Card,
+  ImageContainer,
+  Image,
+  Line,
+  ContentContainer,
+  Header,
+  Title,
+  DateText,
+  Body,
+} from "./index.styles";
+
+function CardComponent({ title, dateAdded, body, imageUrl }) {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const isLongBody = body.length > 200;
+
+  const toggleExpansion = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  return (
+    <Card>
+      <ImageContainer>
+        <Image src={imageUrl} alt="Card Image" />
+        <Line />
+      </ImageContainer>
+      <Line />
+      <ContentContainer>
+        <Header>
+          <Title>{title}</Title>
+          <DateText>{new Date(dateAdded).toLocaleDateString()}</DateText>
+        </Header>
+        <Body expanded={isExpanded}>{body}</Body>
+        {isLongBody && (
+          <Button
+            variant="text"
+            onClick={toggleExpansion}
+            sx={{ p: 0, color: "#3568E5", width: "fit-content" }}
+          >
+            {isExpanded ? "READ LESS" : "READ MORE"}
```

Comment:
The font size on this button is a bit big. Also be sure to bold the font here

Reply:
Thanks for adding the mock data for testing! Is this okay? 
![image](https://github.com/uoftblueprint/tces/assets/73791848/df95789b-7b00-4e91-a18c-6d5108a090e8)